executor/adapter.go

```diff
@@ -245,6 +245,8 @@ func (a *ExecStmt) Exec(ctx context.Context) (sqlexec.RecordSet, error) {
 		// the Projection has two expressions and two columns in the schema, but we should
 		// not return the result of the two expressions.
 		return a.handleNoDelayExecutor(ctx, sctx, e)
+	} else if _, ok := e.(*DDLExec); ok {
```

Comment:
This can also be removed.

Reply:
If remove this, create view return `Empty Set` instead of `Query OK`

Reply:
The CreateView will not reach here after we remove the setSchema

Reply:
Add this comment.